arbor/include/arbor/simulation.hpp

```diff
@@ -51,6 +53,8 @@ class ARB_ARBOR_API simulation {
 
     time_type run(time_type tfinal, time_type dt);
 
+    time_type min_delay();
```

Comment:
would it be useful to add something like `max_epoch_length`, too?

Reply:
True.